src/components/App.js

```diff
@@ -1 +1,23 @@
-// create your App component here
+import React, { useState, useEffect} from "react"
+
+function App() {
+  const [dogImage, setDogImage] = useState(null)
+
+  useEffect(() => {
+    fetch("https://dog.ceo/api/breeds/image/random")
+    .then(resp => resp.json())
+    .then((data) => setDogImage(data.message))
+  }, [])
+
+  if(!dogImage){
+    return <p>Loading...</p>
+  }
+
+  return (
+    <div>
+      <img src={dogImage} alt="A Random Dog"/>
+    </div>
```

Comment:
No need to wrap in a `div` here. 
```suggestion
    <img src={dogImage} alt="A Random Dog"/>
```